api/filters.go

```diff
@@ -124,6 +124,46 @@ func (api *API) createFilter(w http.ResponseWriter, r *http.Request) {
 	api.respond.JSON(ctx, w, http.StatusCreated, resp)
 }
 
+func (api *API) postFilter(w http.ResponseWriter, r *http.Request) {
+
```

Comment:
I have been suggest in past not to leave a blank like at the start of a function

Reply:
ah sorry, my own lint habits. 
resolved